dynamic_rest/viewsets.py

```diff
@@ -337,4 +337,19 @@ def list_related(self, request, pk=None, field_name=None):
 
 
 class DynamicModelViewSet(WithDynamicViewSetMixin, viewsets.ModelViewSet):
-    pass
+
+    ENABLE_BULK_CREATION = dynamic_settings.get('ENABLE_BULK_CREATION', True)
```

Comment:
Hm, I did this in the very same way as the rest of things done: https://github.com/AltSchool/dynamic-rest/blob/master/dynamic_rest/viewsets.py#L70-L71